web3/contract.py

```diff
@@ -1606,11 +1606,11 @@ def factory(cls, class_name: str, **kwargs: Any) -> PropertyCheckingFactory:
 
 class ContractEvent(BaseContractEvent):
     @combomethod
-    def getLogs(
+    def get_logs(
         self,
         argument_filters: Optional[Dict[str, Any]] = None,
-        fromBlock: Optional[BlockIdentifier] = None,
-        toBlock: Optional[BlockIdentifier] = None,
+        from_block: Optional[BlockIdentifier] = None,
```

Comment:
@fselmo just confirming that you meant I should snakecase these keyword args up through line 1674:
`self._get_event_filter_params(
                abi, argument_filters, blockHash, fromBlock=from_block, toBlock=to_block)`

Is this right?

Reply:
Yeah. I think ultimately the `FilterParams` type in _types.py_ should be the only translation layer between snake_case and camelCase. So all the actual python function arguments should be snake-cased, as is convention. This includes `blockHash` in here though which looks like we missed.